src/main/java/andrew/BuildingGame/Game/Game.java

```diff
@@ -220,4 +225,20 @@ private void gameCleanup() {
     }
 //    BGReady.unreadyAllParticipants();
   }
+
+  private Boolean isBuildingRound() {
+    return roundNumber % 2 == 0 && isNeitherInitNorInitPromptState() && !isFinalRound(roundNumber);
```

Comment:
Can this be replaced with the following?

`return gameState == GameStateManager.GameState.BUILD && !isFinalRound(roundNumber)`

Reply:
I tried replacing it with the above, but the final point you mentioned in the pull request regarding 3 build rounds would actually fail.